packages/zipkin/src/instrumentation/httpClient.js

```diff
@@ -0,0 +1,46 @@
+const Annotation = require('../annotation');
+const Request = require('../request');
+
+function requiredArg(name) {
+  throw new Error(`HttpClientInstrumentation: Missing required argument ${name}.`);
+}
+
+class HttpClientInstrumentation {
+  constructor({
+    tracer = requiredArg('tracer')
+  }) {
+    this.tracer = tracer;
+  }
+
+  recordRequest(serviceName, request, remoteServiceName, url, method) {
```

Comment:
the remoteServiceName can also be in the ctor (as this won't change per request even if the associated IP can)

Reply:
👍